remme/tp/account.py

```diff
@@ -110,7 +110,7 @@ def is_address_account_type(self, address):
         """
         return address.startswith(self._prefix) or \
                 address.startswith(NodeAccountHandler()._prefix) or \
-                address == ZERO_ADDRESS
+                address in (ZERO_ADDRESS,)
```

Comment:
What is that for?

Reply:
@eugene-babichenko for the future address of consensus.